virtualcluster/config/sampleswithspec/clusterversion_v1_nodeport.yaml

```diff
@@ -103,9 +103,8 @@ spec:
       kind: Service
       metadata:
         name: etcd
-        annotations:
-          service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
       spec:
+        publishNotReadyAddresses: true
```

Comment:
Do we need/want this for the `clusterversion_v1_loadbalancer.yaml` as well?

Reply:
The file has been changed,  please check the change list. @christopherhein

Reply:
🤦 sorry about that!